app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -121,6 +124,58 @@ fun ApprovalButton(
     }
 }
 
+@Composable
+fun infoButton(modifier: Modifier = Modifier, isEnabled: Boolean,
```

Comment:
Kindly rename this to InfoButton. For composable we want to have the name of the composable functions start with a capital letter while other non-composable functions can start with a small letter.

Reply:
Thanks for highlight. Addressed.!